content/rs/databases/active-active/causal-consistency-crdb.md

```diff
@@ -1,11 +1,17 @@
 ---
-Title: Causal Consistency in an Active-Active Database
-description:
+Title: Enable causal consistency in an Active-Active database
+linkTitle: Enable causal consistency
+description: 
 weight: $weight
 alwaysopen: false
 categories: ["RS"]
+aliases: [
+    /rs/administering/database-operations/causal-consistency-crdb/,
+    /rs/databases/active-active/causal-consistency-crdb.md,
+    /rs/databases/active-active/causal-consistency-crdb/,
+]
 ---
-When you enable Causal Consistency in Active-Active databases,
+When you enable casual consistency in Active-Active databases,
```

Comment:
Casual consistency and causal consistency are different. In this case, causal is the correct term.
```suggestion
When you enable causal consistency in Active-Active databases,
```